samples/Samples.AspNetCoreMvc2/Samples.AspNetCoreMvc2.csproj

```diff
@@ -21,9 +21,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNet.WebApi.Core" Version="5.2.6" />
-    <PackageReference Include="Microsoft.AspNetCore" Version="2.0.4" />
-    <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="2.0.4" />
+    <PackageReference Include="Microsoft.AspNetCore" Version="2.1.3" />
```

Comment:
Let's stick with 2.0.x for now since we don't support .NET Core 2.1 yet.
Looks like the `[ApiController]` was added and 2.1 and we don't need it for this sample.

https://docs.microsoft.com/en-us/aspnet/core/web-api/?view=aspnetcore-2.1#annotate-class-with-apicontrollerattribute

Later we can add another sample app for ASP.NET Core MVC 2.1.